Reference/Templating/Mvc/views.md

```diff
@@ -10,82 +10,56 @@ _Working with MVC Views and Razor syntax in Umbraco_
 
 ## Properties available in Views
 
-All Umbraco views inherit from `Umbraco.Web.Mvc.UmbracoTemplatePage` which exposes many properties that are available in razor:
```

Comment:
I think we need to show the Modelsbuilder approach 'first' as out of the box Modelsbuilder is turned on - but I think we also need to show what happens when Modelsbuilder is turned off - eg the view will inherit from UmbracoViewPage... also you can have partials etc inherit from UmbracoViewPage and still work if Modelsbuilder is turned on...